handbook/engineering/web/index.md

```diff
@@ -88,13 +92,11 @@ After the 20th of each month, we hold a retrospective, to reflect on the past it
 At the beginning of each iteration, the engineering manager will:
 
 - Schedule the retrospective meeting
-- Set up a Slack reminder three days before the retrospective meeting, asking teammates to write their discussion topics in the retrospective document
-
-The meeting notes for all past web team retrospectives can be found [here](https://docs.google.com/document/d/1YW45Dksk0vIn7drhatwLyo6YbMMkS-naHcuShUi1OOw/edit#heading=h.dxt1jy5hsf1d).
+- Set up a Slack reminder three days before the retrospective meeting, asking teammates to write their discussion topics [in our retrospectives document](https://docs.google.com/document/d/1YW45Dksk0vIn7drhatwLyo6YbMMkS-naHcuShUi1OOw/edit#heading=h.dxt1jy5hsf1d)
 
-We do monthly backlog triages together, where we go through the issues in the "To do" column of our [Project board](https://github.com/orgs/sourcegraph/projects/45) and decide which issues we should work on soon by moving them to the "Next" column.
+We do regular backlog triages together, where we go through the issues in our backlog.
```

Comment:
I like that we prior had a link to our backlog – is there still a way to link to it? Is it just the web team tag on the github now (which I think might still be useful to link to)?

Reply:
Good point, added back. It's indeed just the `team/web` label (and there are a bunch of additional self-explanatory labels to narrow the domain). The label is documented under "Contact" BTW (as in, how to file issues on us)